ooniprobe/View/RunTest/TestOverviewViewController.m

```diff
@@ -133,7 +130,7 @@ -(IBAction)run:(id)sender{
 - (void) prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     if ([[segue identifier] isEqualToString:@"toTestSettings"]){
         SettingsTableViewController *vc = (SettingsTableViewController * )segue.destinationViewController;
-        [vc setTestSuite:testSuite];
+        //[vc setTestSuite:testSuite];
```

Comment:
Is this code that we still need to convert or code that we can safely get rid with?

Reply:
There is no time we need to go directly to the settings from the TestOverview page. I am still looking into this.

Reply:
Alright!